components/typography/typography.js

```diff
@@ -55,31 +53,38 @@ if (!head.querySelector('#d2l-typography-font-face')) {
 			font-size: 0.95rem;
 			font-weight: 400;
 			line-height: 1.4rem;
-			/*@apply --d2l-font-custom;*/
 		}
 
-		/*
 		.d2l-typography .d2l-body-standard {
-			@apply --d2l-body-standard-text;
+			font-size: 0.95rem;
+			font-weight: 400;
+			line-height: 1.4rem;
 		}
 
 		.d2l-typography .d2l-body-compact {
-			@apply --d2l-body-compact-text;
+			font-size: 0.8rem;
+			font-weight: 400;
+			line-height: 1.2rem;
 		}
 
 		.d2l-typography .d2l-body-small {
-			@apply --d2l-body-small-text;
+			color: var(--d2l-color-tungsten);
+			font-size: 0.7rem;
+			font-weight: 400;
+			line-height: 1rem;
+			margin: auto;
 		}
 
 		.d2l-typography .d2l-label-text {
-			@apply --d2l-label-text;
+			font-size: 0.7rem;
+			line-height: 1rem;
+			font-weight: 700;
+			letter-spacing: 0.2px;
 		}
-		*/
 
 		.d2l-typography p,
 		.vui-typography p {
 			margin: 1rem 0;
```

Comment:
I wonder when we want to phase out these old `vui-typography` selectors...

Reply:
I think we can probably clean this up.  There's probably some clean-up that can be done - some may already be doubly specifying `d2l-typography` and `vui-typography` but I will put together a PR and update them as needed.  Looks like Thermometer and CopyCat referencing the mixins but I think we should be able to clean up the `vui-typography` class name without affecting them.  Will double-check though.